controllers/helpers.go

```diff
@@ -338,6 +341,64 @@ func GetImage(ctx context.Context, client *prismclientv3.Client, id infrav1.Nuta
 	}
 }
 
+type ImageLookup struct {
+	BaseOS     string
+	K8sVersion string
+}
+
+func GetImageWithFormat(
```

Comment:
I really don't like naming here.

Try to think about GetImageByLookup or something similar which will connect the function with ImageLookup.

Reply:
fair enough.

Reply:
changed the function.